server/config/config.go

```diff
@@ -93,8 +93,11 @@ type Config struct {
 	// be automatically clamped to the range.
 	TSOUpdatePhysicalInterval typeutil.Duration `toml:"tso-update-physical-interval" json:"tso-update-physical-interval"`
 
-	// Local TSO service related configuration.
-	LocalTSO LocalTSOConfig `toml:"local-tso" json:"local-tso"`
+	// EnableLocalTSO is used to enable the Local TSO Allocator feature,
+	// which allows the PD server to generate Local TSO for certain DC-level transactions.
+	// To make this feature meaningful, user has to set the "zone" label for the PD server
+	// to indicate which DC this PD belongs to.
+	EnableLocalTSO bool `toml:"enable-local-tso" json:"enable-local-tso"`
```

Comment:
Incompatible changes need to be confirmed whether it has no effect on downstream.

Reply:
This feature is not released yet and there is no effect on downstream. It's ok to change.